models/gyroscope/gyro_tokens.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+    schema = 'gyroscope',
+    alias = 'gyro_tokens'
```

Comment:
If the schema is `gyroscope`,
then the model name would be `gyroscope_gyro_tokens`.

Reply:
yup, sorry for the oversight